samples/nrf9160/gps/prj.conf

```diff
@@ -5,14 +5,38 @@
 #
```

Comment:
I believe that the sample documentation should be updated as well, to indicate the SUPL library support.

Reply:
I think we need to write the documentation in the first place.